test/functional/testlib.bash

```diff
@@ -961,17 +961,51 @@ bump_format() {
 	new_version="$((version+20))"
 	new_version_path="$env_name"/web-dir/"$new_version"
 	format="$(< "$env_name"/web-dir/"$version"/format)"
-	mom="$new_version_path"/Manifest.MoM
 
 	# create two new versions for the format bump, each version separated by 10
 	# from each other and from the current latest version
 	create_version "$env_name" "$middle_version" "$version" "$format"
 	create_version -r "$env_name" "$new_version" "$middle_version" "$((format+1))"
+
+	# Update the +10 version
+	mom="$middle_version_path"/Manifest.MoM
+
+	# +10 and +20 versions should have matching content
+	sudo rm -rf "$middle_version_path"
+	sudo cp -r "$new_version_path" "$middle_version_path"
+
+	# Update +10 format and versions
+	update_manifest -p "$mom" format "$((format))"
+	update_manifest -p "$mom" version "$middle_version"
+	update_manifest -p "$mom" previous "$version"
+
+	# The +20 version updates the os-release and format files. Change
+	# the versions of these files in the +10 to the +10's version.
+	update_manifest -p "$mom" file-version os-core "$middle_version"
+	update_manifest -p "$middle_version_path"/Manifest.os-core file-version /usr/lib/os-release "$middle_version"
+	update_manifest -p "$middle_version_path"/Manifest.os-core file-version /usr/share/defaults/swupd/format "$middle_version"
+
+	bundles=($(awk '/^M\.\.\./ { print $4 }' "$mom"))
+	IFS=$'\n'
+	for bundle in ${bundles[*]}; do
+		manifest="$middle_version_path"/Manifest."$bundle"
+		update_manifest -p "$manifest" format "$((format))"
+		update_manifest -p "$manifest" version "$middle_version"
+		update_manifest -p "$manifest" previous "$version"
+		sudo rm -rf "$manifest".tar
+		create_tar "$manifest"
```

Comment:
If you make changes to a manifest using the update_manifest function without using the _partial_ flag _-p_ the tar of the manifest will be automatically re-created. So instead of removing and creating the tar here you could just remove the paste _-p_ from the last update_manifest.